p4include/p4d2model_bmss_meta.map

```diff
@@ -0,0 +1,29 @@
+# This file maps p4d2 metadata to the bmv2 simple switch standard and intrinsic metadata
+
+# parser
+parser_input_t.ingress_port       -> standard_metadata.ingress_port
```

Comment:
because simple switch calls it standard_metadata and v1model actually renames standard_metadata_t to standard_metadata with a ```@name``` annotation